save-backend/src/main/resources/application.properties

```diff
@@ -1,3 +1,4 @@
 server.port = 5000
 management.endpoints.web.exposure.include=health,info,prometheus
-spring.liquibase.enabled=false
\ No newline at end of file
+spring.liquibase.enabled=false
+spring.profiles.active=dev
```

Comment:
This shouldn't be activated here, but rather in tests or manually if you are running services locally

Reply:
https://github.com/cqfn/save-cloud/pull/31/commits/c8328125781ebc82757436e6c25c22613b40c85e#r584667392